### ISSUE TITLE
New Rule: Contrast-Focus-Indicator

### ISSUE THREAD

**Issue by jeeyyy** [dequelabs/axe-core#1297]
# Rule/ Check Name: focus-indicator-contrast

The rule tests color contrast between the border/outline and background color for focusable elements, when in focus/ hover states. This is experimental rule.

**Tags:** wcag20, wcag247

## Matcher

Any focusable element.

## Checks

Check focusable elements that have focus/hover styles from CSSOM, and compare color contrast of the border/outline against the background (including any pseudo styles), ignoring elements where outline is not changed, or border is set to none/ 0.

**Return true:** if color contrast is above desired level.
**Return false:** if color contrast is lower than desired level.
**Return needs review:** if contrast ratio cannot be determined.

**Comment by jannakha** [comment]
+1 very useful rule to test!
